app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/fragments/ConversationsFragment.java

```diff
@@ -57,7 +59,10 @@ public void onViewCreated(View view, Bundle savedInstanceState){
         DummyDatabase.getInstance().reset();
 
         for (int i = 0; i < users.length; i++) {
-            Conversation conv = new Conversation(users[i], Arrays.asList(messages[i]));
+            Set<User> aux = new HashSet<>();
```

Comment:
A simple remark, try to make variables names more meaningful to understand them better

Reply:
aux for example